pkg/operator/ceph/webhook-config.go

```diff
@@ -0,0 +1,237 @@
+						Rules: []admv1.RuleWithOperations{
+							{
+								Rule: admv1.Rule{
+									APIGroups:   []string{"ceph.rook.io"},
+									APIVersions: []string{"v1"},
+									Resources:   []string{"cephclusters"},
+								},
+								Operations: []admv1.OperationType{
+									admv1.Update,
+									admv1.Create,
+									admv1.Delete,
+								},
+							},
+						},
+						ClientConfig: admv1.WebhookClientConfig{
+							Service: &admv1.ServiceReference{
+								Name:      admissionControllerAppName,
+								Namespace: namespace,
+								Path:      &serviceCephClusterPath,
+							},
+						},
+						AdmissionReviewVersions: []string{"v1", "v1beta1"},
+						SideEffects:             &sideEffects,
+						TimeoutSeconds:          &timeout,
+					},
+					{
+						Name: fmt.Sprintf("cephblockpool-wh-%s-%s.rook.io", admissionControllerAppName, namespace),
+						Rules: []admv1.RuleWithOperations{
+							{
+								Rule: admv1.Rule{
+									APIGroups:   []string{"ceph.rook.io"},
+									APIVersions: []string{"v1"},
+									Resources:   []string{"cephblockpools"},
+								},
+								Operations: []admv1.OperationType{
+									admv1.Update,
+									admv1.Create,
+									admv1.Delete,
+								},
+							},
+						},
+						ClientConfig: admv1.WebhookClientConfig{
+							Service: &admv1.ServiceReference{
+								Name:      admissionControllerAppName,
+								Namespace: namespace,
+								Path:      &serviceCephBlockPoolPath,
+							},
+						},
+						AdmissionReviewVersions: []string{"v1", "v1beta1"},
+						SideEffects:             &sideEffects,
+						TimeoutSeconds:          &timeout,
+					},
+					{
+						Name: fmt.Sprintf("cephobjectstore-wh-%s-%s.rook.io", admissionControllerAppName, namespace),
+						Rules: []admv1.RuleWithOperations{
+							{
+								Rule: admv1.Rule{
+									APIGroups:   []string{"ceph.rook.io"},
+									APIVersions: []string{"v1"},
+									Resources:   []string{"cephobjectstores"},
+								},
+								Operations: []admv1.OperationType{
+									admv1.Update,
+									admv1.Create,
+									admv1.Delete,
+								},
+							},
+						},
+						ClientConfig: admv1.WebhookClientConfig{
+							Service: &admv1.ServiceReference{
+								Name:      admissionControllerAppName,
+								Namespace: namespace,
+								Path:      &serviceCephObjectStorePath,
+							},
+						},
+						AdmissionReviewVersions: []string{"v1", "v1beta1"},
+						SideEffects:             &sideEffects,
+						TimeoutSeconds:          &timeout,
+					},
+				},
+			}
+
+			_, _ = clusterdContext.Clientset.AdmissionregistrationV1().ValidatingWebhookConfigurations().Create(ctx, validatingWebhook, metav1.CreateOptions{})
+
+			return nil
+		}
+
+	}
+
+	return nil
+}
+
+func deleteIssuerAndCetificate(ctx context.Context, clusterdContext *clusterd.Context) {
+	_ = clusterdContext.Admission.Certificates(namespace).Delete(ctx, certificateName, metav1.DeleteOptions{})
```

Comment:
Instead of ignoring the err, let's at least log any errors

Reply:
logging errors now